python/tvm/tir/schedule/schedule.py

```diff
@@ -540,6 +540,23 @@ def get_consumers(self, block: Union[BlockRV, str]) -> List[BlockRV]:
         block = self._normalize_block_arg(block)
         return list(_ffi_api.ScheduleGetConsumers(self, block))  # type: ignore # pylint: disable=no-member
 
+    @type_checked
+    def get_output_blocks(
+        self,
+        func_name: Optional[str] = None,
+    ) -> List[BlockRV]:
+        """Get the list of output blocks
+        An output block is a block which has atleast one buffer being written
+        to, but is not allocated within the PrimFunc
+
+        Returns
+        -------
+        output_blocks : List[BlockRV]
+            A list of all blocks that write to some output buffer
+
+        """
+        return list(_ffi_api.ScheduleGetOutputBlocks(self, func_name))  # type: ignore # pylint: disable=no-member
```

Comment:
One thing a bit tricky is that the output block is based on the parent scope/block instead of PrimFunc. Would be great to use root block as input if you'd like to get the output blocks of the prim function

Reply:
Thanks for the suggestion. I thought about that, but since the root block does not have to be passed in, I thought of not taking any arguments, but I guess I can modify it to take a block and return output blocks in the scope of that block.

Do you think that makes more sense? We can then call it with the root block if we need all output blocks for the PrimFunc

Reply:
Thanks for your update.
Taking no arguments is a good choice for the root block. And also "take a block and return output blocks in the scope of that block." makes more sense to me. That helps if we have nested blocks.

Reply:
Sure, thanks. I'll update the PR to take a block and return output blocks in that scope.

Reply:
I've updated the PR with the changes. Please take a look now, thanks.